packages/core/src/card/card.element.scss

```diff
@@ -0,0 +1,11 @@
+@import './../styles/tokens/generated/index';
+
+:host {
+  --width: auto;
+  --height: auto;
+
+  --background: #{$cds-alias-object-container-background};
+  --border: #{$cds-alias-object-border-width-100} solid #{$cds-alias-object-container-border-color};
```

Comment:
We had a discussion before with Cory about this one and we end up with the conversation here https://github.com/vmware/clarity/pull/5538#discussion_r580341079 

The `--broder-line` was not part of the APi and we didn't thought about it before. I tried to see how `inherit` `unset`, etc will behave and nothing helped to inherit the initial value we set in the component. I believe switching to the API you are proposing will be beneficial. 

@coryrylan Any thoughts?

Reply:
We typically avoid splitting out shorthand properties like this primarily for maintenance. By exposing the highest level property possible they can override any number of properties without us maintaining a very large number of css properties.

For example we expose `--background` so developers have access to all shorthand values (size, clip, color, img) without maintaining 8 more custom properties for background or having to go back and add them later. Similar patterns can be found for `--padding` and `--margin`. We found early on it became hard to maintain and document so many property values. It may be more work to re-set defaults but we should encourage use of the theme aliases instead of component level properties when possible.

I do think we could add a new object border token to represent the style value at the system level.

```css
:root {
  --cds-alias-object-border-style: solid;
}

:host {
  --border: #{$cds-alias-object-border-width-100} #{$cds-alias-object-border-style} #{$cds-alias-object-container-border-color};
}
```